### PR TITLE
Retry opening DB connections.

### PR DESCRIPTION
**Pull request by alexeyklyukin**
Avoid calling ping from the closed connection.

### CODE REVIEW

File: pkg/cluster/pg.go [hunk 1]
```diff
@@ -52,33 +52,33 @@ func (c *Cluster) initDbConn() error {
...
 
-	if err != nil {
-		return fmt.Errorf("could not init db connection: %v", err)
+			if err2 := conn.Close(); err2 != nil {
+				c.logger.Errorf("error when closing PostgreSQL connection after another error: %v", err)
+				return false, err2
+			}
+
+			return false, err
+		}); finalerr != nil {
```
Comment (ikitiki) [inline pkg/cluster/pg.go]: looks messy

**Comment by ikitiki** [comment]
👍

**Comment by ikitiki** [comment]
👍

**Comment by alexeyklyukin** [comment]
👍

**Pull request merged**